DEPLOYMENT.md

```diff
@@ -0,0 +1,57 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wild card SSL certificate be installed on the OpenShift router. However, in situations where this is not desirable, RHMAP can be exposed using single URL.
+
+## RHMAP Proxy
+
+RHMAP Proxy is a separate component which should be deployed to OpenShift per MBaaS project. Its purpose is to route requests that are not using the standard URL scheme. The RHMAP Proxy application is bundled within the RHMAP RPM in the form of an OpenShift template. Creating this application will create an OpenShift route which will expose this service. Carry out the following steps to configure RHMAP Core to use non-wildcard routing.
```

Comment:
" Its purpose is to route requests that are not using the standard URL scheme." 
That sentence seems to be too generic and do not provide enough information.

I would say something like this:
`Proxy would route from single external host requests to cloud apps and RHMAP core instances deployed on openshift. For example:

.. Put examples here
`